beacon_node/execution_layer/src/lib.rs

```diff
@@ -1402,20 +1510,225 @@ impl<T: EthSpec> ExecutionLayer<T> {
         debug!(
             self.log(),
             "Sending block to builder";
-            "root" => ?block_root,
+            "root" => %block_root,
         );
+
         if let Some(builder) = self.builder() {
-            builder
-                .post_builder_blinded_blocks(block)
-                .await
-                .map_err(Error::Builder)
-                .map(|d| d.data)
+            let (payload_result, duration) =
+                timed_future(metrics::POST_BLINDED_PAYLOAD_BUILDER, async {
+                    builder
+                        .post_builder_blinded_blocks(block)
+                        .await
+                        .map_err(Error::Builder)
+                        .map(|d| d.data)
+                })
+                .await;
+
+            match &payload_result {
+                Ok(payload) => {
+                    metrics::inc_counter_vec(
+                        &metrics::EXECUTION_LAYER_BUILDER_REVEAL_PAYLOAD_OUTCOME,
+                        &[metrics::SUCCESS],
+                    );
+                    info!(
+                        self.log(),
+                        "Builder successfully revealed payload";
+                        "relay_response_ms" => duration.as_millis(),
+                        "block_root" => %block_root,
+                        "fee_recipient" => %payload.fee_recipient,
+                        "block_hash" => %payload.block_hash,
+                        "parent_hash" => %payload.parent_hash
+                    )
+                }
+                Err(e) => {
+                    metrics::inc_counter_vec(
+                        &metrics::EXECUTION_LAYER_BUILDER_REVEAL_PAYLOAD_OUTCOME,
+                        &[metrics::FAILURE],
+                    );
+                    crit!(
```

Comment:
Maybe an error seeing as this is kinda outside our control, and maybe not that severe?

```suggestion
                    error!(
```

Reply:
My understanding of real-world events so far is that if a builder isn't revealing blocks to the BN then those blocks are lost and the relay is probably bugged. Given that, I'd be tempted to start with a `crit` here and then perhaps relax it to an `error` if we see that builders are failing to reveal to us but actually publishing on the network. I think this is particularly relevant whilst we don't have tracking of missed blocks in the validator manager.

Reply:
> the relay is probably bugged

... and probably not going to do anything about it for several hours 😅

Reply:
Yeah sounds good. I thought there'd been at least one incident where the blocks were getting broadcast but not revealed to the BN, but I don't have a source. Happy to leave it as a CRIT.